apis/Google.Cloud.Diagnostics.AspNetCore/Google.Cloud.Diagnostics.AspNetCore/Trace/CloudTraceExtension.cs

```diff
@@ -153,13 +153,11 @@ internal static TraceHeaderContext CreateTraceHeaderContext(IServiceProvider pro
 
         /// <summary>
         /// Creates a singleton <see cref="IManagedTracer"/> that is a <see cref="DelegatingTracer"/>.
-        /// The <see cref="DelegatingTracer"/> will use an <see cref="IHttpContextAccessor"/> under the hood
-        /// to get the current tracer which is set by the <see cref="ContextTracerManager"/>.
+        /// The <see cref="DelegatingTracer"/> will get the current tracer which is set by the <see cref="ContextTracerManager"/>.
         /// </summary>
         internal static IManagedTracer CreateManagedTracer(IServiceProvider provider)
         {
-            var accessor = provider.GetServiceCheckNotNull<IHttpContextAccessor>();
-            return ManagedTracer.CreateDelegatingTracer(() => ContextTracerManager.GetCurrentTracer(accessor));
+            return ManagedTracer.CreateDelegatingTracer(() => ContextTracerManager.GetCurrentTracer());
```

Comment:
suggestion:

```
internal static IManagedTracer CreateManagedTracer() =>
    ManagedTracer.CreateDelegatingTracer(() => ContextTracerManager.GetCurrentTracer());
```

Or just remove this function all together now that the logic is so simple.

Reply:
Yes, I considered doing that too, but didn't want to touch the original code too much. Removing it altogether definitely makes sense.